R/check_folder.R

```diff
@@ -0,0 +1,256 @@
+#' Check the folder structure
+#'
+#' For the time being, this function only checks projects.
+#' Keep in mind that R packages have requirements for the folder structure.
+#' That is one of things that `check_cran()` checks.
+#'
+#' # Recommended folder structure
+#'
+#' - `source`: contains all `R` scripts and `Rmd` files.
+#' - `data`: contains all data files.
+#'
+#' # `source`
+#'
+#' A simple project with only `R` scripts or only `Rmd` files can place all the
+#' files directly in the `source` folder.
+#'
+#' More elaborate projects should place in the files in several folders under
+#' `source`.
+#' Place every `bookdown` document in a dedicated folder.
+#' And create an RStudio project for that folder.
+#'
+#' # `data`
+#'
+#' Simple projects in which `source` has no sub folders, place `data` at the
```

Comment:
```suggestion
#' Simple projects in which `source` has no subfolders, place `data` at the
```

Reply:
also further in this document, I will leave it to you if you want to write this as sub folders or subfolders